jobs/opensearch_dashboards/templates/config/opensearch_dashboards.conf.erb

```diff
@@ -49,15 +49,15 @@ opensearchDashboards.defaultAppId: <%= p('opensearch_dashboards.defaultAppId') %
 # Branding for opensearch
 opensearchDashboards.branding:
   logo:
-    defaultUrl: "https://raw.githubusercontent.com/cloud-gov/opensearch-boshrelease/refs/heads/branding/src/images/cloud-gov%20logomark.svg"
+    defaultUrl: "https://raw.githubusercontent.com/cloud-gov/opensearch-boshrelease/refs/heads/main/src/images/cloud-gov%20logomark.svg"
     darkModeUrl: ""
   mark:
-    defaultUrl: "https://raw.githubusercontent.com/cloud-gov/opensearch-boshrelease/refs/heads/branding/src/images/cloud-gov%20logomark.svg"
-    darkModeUrl: "https://raw.githubusercontent.com/cloud-gov/opensearch-boshrelease/496076e1ab5db4afe31bb29bbee54aaac602f4b1/src/images/cloud-gov%20logomark.svg"
+    defaultUrl: "https://raw.githubusercontent.com/cloud-gov/opensearch-boshrelease/refs/heads/main/src/images/cloud-gov%20logomark.svg"
+    darkModeUrl: "https://raw.githubusercontent.com/cloud-gov/opensearch-boshrelease/refs/heads/main/src/images/cloud-gov%20logomark.svg"
```

Comment:
Should we set the dark mode URL? I haven't checked to see what it looks like in dark mode

Reply:
Just checked it out and it looks okay, the contrast between our black on the logo and the darkmode isn't horrible